src/lib/uTop_main.ml

```diff
@@ -1372,6 +1372,9 @@ let preload = ref []
 
 let prepare () =
   Toploop.set_paths ();
+#if OCAML_VERSION >= (5, 0, 0)
+  Load_path.(init ~auto_include:(fun _ _ -> raise Not_found) (get_paths ()));
+#endif
```

Comment:
If we want to change the API, personally I like
```suggestion
  Toploop.set_paths ~auto_include:None ();
```
or
```suggestion
  Toploop.set_paths ~auto_include:(fun _ _ -> None) ();
```
instead of using exceptions.

Reply:
There isn't any choice about that part of the API, I'm afraid - that function is hooking into the behaviour of `Load_path.find` and `Load_path.find_uncap` both of which raise `Not_found` on a miss.